styles/portfolio.css

```diff
@@ -0,0 +1,71 @@
+.sidebar {
+    grid-area: sidebar;
+    text-align: center;
+}
+
+.content {
+    grid-area: content;
+}
+
+.header {
+    grid-area: header;
+}
+
+.footer {
+    grid-area: footer;
+}
+
+.wrapper {
+    background-color: #fff;
+    color: #444;
+}
+
+.wrapper {
```

Comment:
Nice use of grid layout for the parts of the portfolio page, and then using media queries to reflow them when the screen gets too narrow (like for a mobile device).